Api/KanbanizeApiV1.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+namespace Chaplean\Bundle\KanbanizeApiClientBundle\Api;
+
+use Chaplean\Bundle\ApiClientBundle\Api\AbstractApi;
+use Chaplean\Bundle\ApiClientBundle\Api\Parameter;
+use Chaplean\Bundle\ApiClientBundle\Api\Route;
+use GuzzleHttp\ClientInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+/**
+ * Class KanbanizeApiV1.
+ *
+ * @method Route getProjectsAndBoards()   Get projects and boards     ( https://kanbanize.com/api/#get_projects_and_boards )
+ * @method Route postBoardStructure()     Get board structure         ( https://kanbanize.com/api/#get_board_structure )
+ * @method Route postFullBoardStructure() Get full board structure    ( https://kanbanize.com/api/#get_full_board_structure )
+ * @method Route postBoardSettings()      Get board settings          ( https://kanbanize.com/api/#get_board_settings )
+ * @method Route postBoardActivities()    Get board activities        ( https://kanbanize.com/api/#get_board_activities )
+ * @method Route postCreateTask()         Create a new task           ( https://kanbanize.com/api/#create_new_task )
+ * @method Route postTaskDetails()        Get details for one task    ( https://kanbanize.com/api/#get_task_details )
+ * @method Route postTasksDetails()       Get details for many task   ( https://kanbanize.com/api/#get_task_details )
+ * @method Route postAllTasks()           Get all tasks for one board ( https://kanbanize.com/api/#get_all_tasks )
+ * @method Route postLogTimeActivities()  Create a new task           ( https://kanbanize.com/api/#get_log_time_activities )
+ *
+ * @package   Chaplean\Bundle\KanbanizeApiClientBundle\Api
+ * @author    Valentin - Chaplean <valentin@chaplean.coop>
+ * @copyright 2014 - 2018 Chaplean (http://www.chaplean.coop)
+ */
+class KanbanizeApiV1 extends AbstractApi
+{
+    /** @var string */
+    private $apikey;
+
+    /** @var string */
+    private $urlPrefix;
+
+    /**
+     * KanbanizeApiV1 constructor.
+     *
+     * @param ClientInterface          $client
+     * @param EventDispatcherInterface $eventDispatcher
+     * @param string                   $urlPrefix
+     * @param string                   $apikey
+     */
+    public function __construct(ClientInterface $client, EventDispatcherInterface $eventDispatcher, string $urlPrefix, string $apikey)
```

Comment:
Return typehinting missing

Reply:
Return typehinting on the constructor isn't allowed